components/molecules/CallToAction.js

```diff
@@ -2,65 +2,81 @@ import PropTypes from "prop-types";
 import { ActionButton } from "../atoms/ActionButton";
 import Link from "next/link";
 import { useTranslation } from "next-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faUser } from "@fortawesome/free-regular-svg-icons";
 
 /**
  * A section that will have a title, small description, and a link to some action we want to user to make
  */
 export function CallToAction(props) {
   const { t } = useTranslation("common");
   return (
-    <aside>
-      <div
-        className={`${
-          props.feedbackActive ? "bg-custom-blue-blue" : "bg-circle-color"
-        } text-white`}
-      >
-        <div className="layout-container pb-10 pt-10 text-sm md:text-base">
-          <h2>{props.title}</h2>
-          <div className="flex flex-col lg:grid lg:grid-cols-2 lg:gap-24 gap-5">
-            {props.description ? (
-              <p className="whitespace-pre-line">{props.description}</p>
-            ) : (
-              <div
-                className="whitespace-pre-line"
-                dangerouslySetInnerHTML={{ __html: props.html }}
+    <div className="bg-[#ebf2fc] text-[#333]">
+      <div className="layout-container pb-10 pt-10 text-sm md:text-base">
+        <div>
+          <div className="flex flex-col">
+            <div className="flex -my-2">
```

Comment:
This y margin isn't being applied due to the preceding "-"

Reply:
This is intended, see https://tailwindcss.com/docs/margin#using-negative-values

Reply:
Oh well then TIL!